src/app/google-analytics-events.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { environment } from '../environments/environment';
+
+declare let ga: any;
+
+@Injectable({
+    providedIn: 'root'
+})
+export class GoogleAnalyticsEventsService {
+
+    emitEvent(
+        eventCategory: string,
+        eventAction: string,
+        eventLabel: string) {
+        if(!environment.production) return;
+
+        if(ga && ga.getAll) {
```

Comment:
It was `if(window.ga && window.ga.getAll)`, I think we had some issues with global variables. @babich-a do you remember details?

Reply:
I do not. But I like using of the `window` in any way more than `declare let ga: any;`